### ISSUE TITLE
Clicking on a reaction button results in a white activation box

### ISSUE THREAD

**Issue by tambry** [StylishThemes/GitHub-Dark#572]
* **Browser**: Chrome 63.0.3239.84 64-bit
* **Operating System**: Widows 10 Pro N 1709 16299.98
* [Screenshot](https://i.imgur.com/05I8hKz.png)

HTML of the section where the issue occurs:
````html
<button type="submit" class="btn-link add-reactions-options-item js-reaction-option-item" data-reaction-label="+1" name="input[content]" aria-label="React with thumbs up emoji" value="THUMBS_UP react">
	<g-emoji alias="+1" class="emoji" fallback-src="https://assets-cdn.github.com/images/icons/emoji/unicode/1f44d.png" ios-version="6.0">👍</g-emoji>
</button>
````

Clicking on ("activating") a reaction button will result in a white box appearing around the emoji.

**Comment by silverwind** [comment]
This is already fixed as of https://github.com/StylishThemes/GitHub-Dark/commit/4797b0a1f9ef9319ca6c3094fe564d2c3677b04c.

**Comment by tambry** [comment]
In that case it might be worth pushing out an update.

**Comment by silverwind** [comment]
Released 1.20.5.
